diff -urN libgdamm-3.0.0/configure.in libgdamm-3.0.0.new/configure.in
--- libgdamm-3.0.0/configure.in	2008-03-16 14:42:15.000000000 +0100
+++ libgdamm-3.0.0.new/configure.in	2008-07-10 17:24:29.413363507 +0200
@@ -34,7 +34,7 @@
 # CURRENT : REVISION : AGE
 LIBGDAMM_SO_VERSION=10:0:0
 
-AC_SUBST(LIBGDAMM_VERSION)
+AC_SUBST([LIBGDAMM_VERSION])
 AC_SUBST(LIBGDAMM_RELEASE)
 AC_SUBST(LIBGDAMM_SO_VERSION)
 
